packages/server/src/app/api/auth/auth.service.ts

```diff
@@ -5,6 +5,7 @@ import { verify, argon2id } from 'argon2';
 import { User, FrontendUser, ChangePassword, ChangeProfile, ChangeEmail, ChangeUsername, UpdateTagline } from '@pulp-fiction/models/users';
 import { UsersService } from '../../db/users/users.service';
 import { JwtPayload } from '@pulp-fiction/models/auth';
+import { String } from 'aws-sdk/clients/appstream';
```

Comment:
wait, for real?

Reply:
pfft so that's where it kept grabbing `String` instead of `string` from